.github/workflows/triage.yml

```diff
@@ -0,0 +1,18 @@
+name: Triage
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-ticket-to-project:
+    name: Core Development project
+    uses: maliput/.github/.github/workflows/triage.yml@main
+    with:
+      number_of_project: "1"
```

Comment:
I had thought about creating another project for this effort (number 2), however I consider that it would be great to have it in the main board. We can filter things out using the repos, labels ,etc. So I would leave it as it is.

Reply:
Set it to 2 to use the "Speed up maliput backend creation" project

Reply:
I'll leave it as is then.